test/streams/streams_channel_test.rb

```diff
@@ -190,6 +191,47 @@ class Turbo::StreamsChannelTest < ActionCable::Channel::TestCase
     end
   end
 
+  test "broadcasting action later with ActiveModel array target" do
+    options = { partial: "messages/message", locals: { message: "hello!" } }
+
+
+    class ::MyModel
```

Comment:
Have you tried re-using the existing `Message` model declared elsewhere in the test suite? It's [ActiveModel::Conversion](https://edgeapi.rubyonrails.org/classes/ActiveModel/Conversion.html)-compliant:

```diff
diff --git a/test/streams/streams_channel_test.rb b/test/streams/streams_channel_test.rb
index f212266..5d47345 100644
--- a/test/streams/streams_channel_test.rb
+++ b/test/streams/streams_channel_test.rb
@@ -1,6 +1,5 @@
 require "test_helper"
 require "action_cable"
-require "active_model"
 
 class Turbo::StreamsChannelTest < ActionCable::Channel::TestCase
   include ActiveJob::TestHelper, Turbo::Streams::ActionHelper
@@ -194,15 +193,9 @@ class Turbo::StreamsChannelTest < ActionCable::Channel::TestCase
   test "broadcasting action later with ActiveModel array target" do
     options = { partial: "messages/message", locals: { message: "hello!" } }
 
-
-    class ::MyModel
-      include ActiveModel::Model
-      attr_accessor :id
-    end
-
-    my_model = MyModel.new(id: 42)
-    target = [my_model, "opt"]
-    expected_target = "opt_my_model_42"
+    message = Message.new(id: 42)
+    target = [message, "opt"]
+    expected_target = "opt_message_42"
 
     assert_broadcast_on "stream", turbo_stream_action_tag("prepend", target: expected_target, template: render(options)) do
       perform_enqueued_jobs do
@@ -215,15 +208,9 @@ class Turbo::StreamsChannelTest < ActionCable::Channel::TestCase
   test "broadcasting action later with multiple ActiveModel targets" do
     options = { partial: "messages/message", locals: { message: "hello!" } }
 
-
-    class ::MyModel
-      include ActiveModel::Model
-      attr_accessor :id
-    end
-
-    one = MyModel.new(id: 1)
+    one = Message.new(id: 1)
     targets = [one, "messages"]
-    expected_targets = "#messages_my_model_1"
+    expected_targets = "#messages_message_1"
 
     assert_broadcast_on "stream", turbo_stream_action_tag("prepend", targets: expected_targets, template: render(options)) do
       perform_enqueued_jobs do

```

Reply:
Ohhh thanks, that's nice, I wasn't looking hard enough :)